src/libpocketsphinx/pocketsphinx.c

```diff
@@ -1176,13 +1176,15 @@ ps_end_utt(ps_decoder_t *ps)
         	int sf, ef;
         	int32 post, lscr, ascr, lback;
 
-        	word = ps_seg_word(seg);
-        	ps_seg_frames(seg, &sf, &ef);
-        	post = ps_seg_prob(seg, &ascr, &lscr, &lback);
-        	E_INFO_NOFN("%-20s %-5d %-5d %-1.3f %-10d %-10d %-3d\n",
-                    	    word, sf, ef, logmath_exp(ps_get_logmath(ps), post),
-                    	ascr, lscr, lback);
-    	    }
+            if(seg != NULL) {
```

Comment:
I'm sorry, beside indentation issues, I'm not sure why seg could be NULL here, it is already checked in for loop.

Reply:
This is I'm sorry, it's seems like I was very glad after fix swig interface and make some hasty conclusion. I rechecked again and again and can't reproduce it today so there is no problem here